app/views/todos/show.html.haml

```diff
@@ -1,7 +0,0 @@
-.container
```

Comment:
I don't think we should remove this. You will get an app error if you got to <navi_url>:3000/todos/#